.travis.yml

```diff
@@ -36,14 +36,16 @@ jobs:
       script: bash scripts/travis-test.sh valkyrie
     - name: reaper
       script: bash scripts/travis-test.sh reaper
+    - name: estuary
+      script: bash scripts/travis-test.sh estuary
     - stage: publish
       name: development
       if: branch == master && type != pull_request
       script: bash scripts/travis-publish.sh
     - name: release
-      if: tag =~ /^(andi|reaper|valkyrie|forklift|flair)@\d{1,3}\.\d{1,3}\.\d{1,3}$/
+      if: tag =~ /^(andi|reaper|valkyrie|forklift|flair|estuary)@\d{1,3}\.\d{1,3}\.\d{1,3}$/
       script: bash scripts/travis-publish.sh
 branches:
   only:
   - master
-  - /(andi|reaper|valkyrie|forklift|flair)@\d{1,3}\.\d{1,3}\.\d{1,3}/
+  - /(andi|reaper|valkyrie|forklift|flair|estuary)@\d{1,3}\.\d{1,3}\.\d{1,3}/
```

Comment:
I don't think Travis will build our branch with the current naming convention that we have used (estuary_umbrella). For travis to actually run I believe that we'll have to rename our branch to `estuary@\d{1,3}\.\d{1,3}\.\d{1,3}`